packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -52,7 +60,15 @@ export function Previewer({
           />
         </UploadDropZone>
         <Text fontWeight="bold">
-          {files.length} {translate('files selected')}
+          {isSuccess ? (
+            <>
+              {uploadedFiles().length} {translate('files uploaded')}
+            </>
+          ) : (
+            <>
+              {remainingFiles().length} {translate('files selected')}
+            </>
+          )}
```

Comment:
Without it React won't compile, I get this error `ERROR: Expected "{" but found "."`

Reply:
try `uploadedFiles().length translate('files uploaded')`

Reply:
As mentioned in the PR, I would rethink these translations because word orderings are different between languages. 

For example, "3 files selected" roughly translates to "파일 3개 선택됨" in korean, of which you notice "3" is now pulled into the middle. We should eventually something like `i18next` to have translation with templates, but for now, I do not feel safe making this half translations. What do you think?

Reply:
That said, I do not what the short term solution could be.  

(1) we could split off strings like you did:`{uploadedFiles().length} {translate('files uploaded')}`
but this doesn't work for all languages.

(2) we could translate them as a whole... <code>translate(`${uploadedFiles.length files uploaded}`)</code>`
but this is a dynamic string 😢 

For long term solution, we should definitely consider using something like `i18next` so that we can use templates in translations.